aiida/tools/dbexporters/tcod.py

```diff
@@ -385,16 +385,18 @@ def _inline_to_standalone_script(calc):
     args = ["{}=load_node('{}')".format(x, input_dict[x].uuid)
             for x in input_dict.keys()]
     args_string = ",\n    ".join(sorted(args))
+    code_string = calc.get_attr('source_file').encode('utf-8')
+    if calc.get_attr('namespace', '__main__').startswith('aiida.'):
+        code_string = "from {} import {}".format(calc.get_attr('namespace', '__main__'),
+                                                 calc.get_attr('function_name','f'))
     return """#!/usr/bin/env runaiida
```

Comment:
This means that in the general case it will write `from __main__ import f`? Is this the expected behaviour?

Reply:
It should not: in general case the condition is ``False`` (``"__main__".startswith('aiida.') == False``), therefore nothing is written. I added ``"__main__"`` just to ignore missing ``namespace``  attribute.